scalajslib/src/ScalaJSModule.scala

```diff
@@ -169,8 +169,10 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
       }
     }
   }
-  override def scalaLibraryIvyDeps = T {
-    super.scalaLibraryIvyDeps() ++ Seq(
+
+  /** Adds the Scala.js Library as mandatory dependency. */
+  override def mandatoryIvyDeps = T {
+    super.mandatoryIvyDeps() ++ Seq(
```

Comment:
This introduces a behavioural change, since now `scalaLibraryIvyDeps` don't contain the scala-js library anymore. I wonder if it makes sense to introduce a breaking change and remove the `scalaLibraryIvyDeps`target all together.

What do you think?

Reply:
At least the target now returns what it claims with its name. ;-) I was under the impression this target is only used internally, but you're right, it's public after all. We should probably document that change. We could also add an additonal `scalaJSLibraryIvyDeps` which is then re-used in `mandatoryIvyDeps`.

Reply:
> We should probably document that change.

Agree! 👍 

> We could also add an additional scalaJSLibraryIvyDeps which is then re-used in mandatoryIvyDeps.

My usual take, let's wait for someone asking for it. I'm more on removing or deprecating `scalaLibraryIvyDeps` than add more of them to make it harder to remove. I would not change anything for now so we can think about it.

Reply:
I added a changelog entry and keep implementation as is.